share/migrations/0038_trust_system_user.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2017-06-03 12:05
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations
+
+
+def trust_system_user(apps, schema_editor):
+    ShareUser = apps.get_model('share', 'ShareUser')
+    Source = apps.get_model('share', 'Source')
+
+    system_user = ShareUser.objects.get(username=settings.APPLICATION_USERNAME)
+    system_user.is_trusted = True
+    system_user.save()
+
+    Source.objects.update_or_create(
```

Comment:
It doesn't have an icon, which I think is enough to exclude it from all the lists (is [this](https://github.com/CenterForOpenScience/SHARE/blob/develop/api/views/workflow.py#L45) the only place we list sources?)